modules/apps/liferay-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,115 @@
+
+body {
+	font-family: Helvetica, Arial, sans-serif;
+	margin: 0;
+}
+
+.sign-in {
+	border: 2px solid #000;
+	border-radius: 200px;
+	color: #000;
+	display: block;
+	float: right;
+	margin-right: 15px;
+	padding: 6px;
+	text-transform: uppercase;
+}
+
+.header-banner {
+	margin-bottom: 80px;
+	padding-top: 90px;
+}
+
+.logo-image {
+	display: block;
```

Comment:
We should probably add some max-width/max-height to control the size of the image even if it is the default one.